components/icon/workflow-icons.css

```diff
@@ -15,28 +15,6 @@ governing permissions and limitations under the License.
   --spectrum-icon-size-m: var(--spectrum-alias-workflow-icon-size-m);
   --spectrum-icon-size-l: var(--spectrum-alias-workflow-icon-size-l);
   --spectrum-icon-size-xl: var(--spectrum-alias-workflow-icon-size-xl);
-  --spectrum-icon-size-xxs: calc(var(--spectrum-icon-size-s) / 2);
-  --spectrum-icon-size-xs: calc(var(--spectrum-icon-size-m) / 2);
-  --spectrum-icon-size-xxl: calc(var(--spectrum-icon-size-m) * 3);
-}
-
-/* Hey! Icon sizes changed recently, double check sizing */
-.spectrum-Icon--sizeXXS {
-  &,
-  img,
-  svg {
-    block-size: var(--spectrum-icon-size-xxs);
-    inline-size: var(--spectrum-icon-size-xxs);
-  }
-}
-
-.spectrum-Icon--sizeXS {
-  &,
-  img,
-  svg {
-    block-size: var(--spectrum-icon-size-xs);
-    inline-size: var(--spectrum-icon-size-xs);
-  }
```

Comment:
We don't have to do this in this PR, but I figured why not drop the unsupported icon sizes? @jianliao thoughts?

Reply:
Ah, I dropped them in my original UI icon t-shirt sizes PR. But I saw @GarthDB added them back, so I added them back, too.

Reply:
Hmm, @GarthDB should we kill them or leave them?

Reply:
Have these tokens not been killed in DNA yet? @jianliao do you know what version of DNA removes them?

Reply:
They are not included in 6.0.0-beta.20. But seems like DNA never provides them, we calculated them out based on medium and small.

Reply:
Right, so they've never been in DNA, but we've always provided those non-standard icon sizes as a holdover from Coral. We should kill them, but lemme know if you think a different PR is a better place for it @GarthDB.